rust-1.55/README.md

```diff
@@ -0,0 +1,47 @@
+# 📧 Unnamed function
+
```

Comment:
Let's remove tar.gz file

Reply:
Yeah, let's remove that.. It would add more complexity to 0.13.. This could be implemented as future iteration.

Reply:
removed 👍